source/exercises100.ktx

```diff
@@ -778,16 +778,17 @@ Z = np.random.randint(0,3,(3,10))
 print((~Z.any(axis=0)).any())
 
 < q61
-Find the nearest value from a given value in an array (★★☆)
+Find the 3 nearest values in order from a given value in an array (★★☆)
 
 < h61
-hint: np.abs, argmin, flat
+hint: np.abs, argpartition, flat, argsort
 
 < a61
-Z = np.random.uniform(0,1,10)
+Z = np.random.uniform(0, 1, 10)
 z = 0.5
-m = Z.flat[np.abs(Z - z).argmin()]
-print(m)
+m = Z.flat[np.abs(Z - z).ravel().argpartition(3)[:3]]
```

Comment:
I would prefere a one line expression such that people can see directly what is the interesting part. I know that it won't work for all kind of specific cases but the goal here is to highlight argpartition.

`print( np.abs(Z - z).argpartition(3)[:3]`